src/JuliennedArrays.jl

```diff
@@ -3,6 +3,9 @@ module JuliennedArrays
 import Base: axes, getindex, setindex!, size
 using Base: @pure, tail
 
+export Slices, Align
+export True, False
```

Comment:
Actually, I think these names are too common to be exported.

Reply:
Who uses them?

Reply:
`Slice` exists in Julia Base with a completely different meaning, I feel that's quite ambiguous to have `Slices` meaning the others. `SliceView` and `AlignView` are more accurate IMO.

`True`/`False` are too common names here and I think they should be either implementation details or preserved by Base. But apparently, people just invent their own without much effort:

https://github.com/SciML/Static.jl/blob/0f293e94fcbfbc812cdeaa796bd549b4ec2bc1ce/src/bool.jl#L2-L11